src/main/java/net/tridentsdk/base/Enchantment.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Trident - A Multithreaded Server Alternative
+ * Copyright 2014 The TridentSDK Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package net.tridentsdk.base;
+
+import java.util.Map;
+
+import com.google.common.collect.Maps;
+
+public final class Enchantment {
+
```

Comment:
The whitespace here does not fit the stuleguidelines. Please read the CONTRIBUTING.md for more information.